modules/ROOT/pages/mule-message-encryption-processor.adoc

```diff
@@ -183,6 +185,7 @@ image::xml-encrypter.png[xml_encrypter]
 |*doc:name* |  |A display name for the element in Studio's Visual Editor. Not applicable for Standalone.
 |*using* |x |XML_ENCRYPTER
 |*input-ref* |  |Enter a Mule expression to define the part(s) of the message Mule should encrypt. If no value is entered, Mule encrypts the entire message payload.
+| *streamName* |  |
```

Comment:
```suggestion
| *streamName* | name of the decrypted payload in a PGP encrypted file. In case a file is encrypted using Mule, this shall help the user keep the original filename so that it is recovered on decryption of the resulting PGP file |
```

Reply:
@aiannucci Can you help with the definition of streamName or recommend someone who can? I'm hoping for something more that “the name of the stream” so people know what it’s for, when to use, and whether it’s required also. Thanks! :)